.github/workflows/release-phar.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  release:
+    types:
+      - created
+
+name: Attach PHAR to release
+
+jobs:
+  build:
+    name: Compile and upload Phar
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+            fetch-depth: 0
+
+      - name: Set PHP 7.3
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.3'
+
+      -
+        name: "Composer install"
+        uses: "ramsey/composer-install@v1"
+        with:
+            composer-options: "--no-scripts --no-dev"
+
+      - name: Build PHAR
+        run: composer compile
+
+      - name: Check existence of compiled .phar
+        run: test -e my127ws.phar && exit 0 || exit 10
+
+      - name: "Upload PHAR to Release"
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: my127ws.phar
+          asset_name: my127ws.phar
```

Comment:
note it will attach as `my127ws.phar` vs. `ws` as with previous releases. Not sure if we care.

Reply:
Can we name it `ws` or do we want to handle that in the README instructions?

Reply:
i updated to rename it to `ws`